# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/GtkCompositorWidget.cpp
# Commit: a264ff9e9f6f
# Full Hash: a264ff9e9f6f87ca0520a884b29c4be90001533e
# Author: stransky <stransky@redhat.com>
# Date: 2024-07-10 21:22:09
# Regressor Bug: 1898476
# File Overlap Count: 1
# Description:
#   Bug 1898476 [Wayland] Provide surface lock by GtkCompositorWidget r=emilio
#   
#   Depends on D214883
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D214884
# ==============================================================================

diff -r f9323daf7abe -r a264ff9e9f6f widget/gtk/GtkCompositorWidget.cpp
--- a/widget/gtk/GtkCompositorWidget.cpp	Wed Jul 10 10:59:52 2024 +0000
+++ b/widget/gtk/GtkCompositorWidget.cpp	Wed Jul 10 10:59:53 2024 +0000
@@ -211,5 +211,9 @@
 }
 #endif
 
+UniquePtr<MozContainerSurfaceLock> GtkCompositorWidget::LockSurface() {
+  return mWidget->LockSurface();
+}
+
 }  // namespace widget
 }  // namespace mozilla